RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -196,6 +196,7 @@
      src = 'jetCoreRegionalStepTracks',
      qualityCuts = [0.6, 0.7, 0.8],
 ))
+(trackingPhase1 & fastSim).toModify(jetCoreRegionalStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
```

Comment:
this could have been simply ```fastSim.toModify``` with a matching removal of L190.
The final result is OK.